.php-cs-fixer.dist.php

```diff
@@ -9,6 +9,7 @@
     ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
+        'trailing_comma_in_multiline' => false,
```

Comment:
i would prefer to fix this rather than diverge from the symfony ruleset.

but as you mentioned in https://github.com/php-http/client-common/pull/247 it would make sense to pin the cs fixer to a specific version to avoid random unrelated cs failures in merge requests.

Reply:
We have to do this because the syntax is not compatible with PHP 7.